apollo-router/src/configuration/cors.rs

```diff
@@ -198,6 +665,32 @@ impl Cors {
                     );
                 }
             }
+
+            // Check per-policy fields for wildcards when credentials are enabled
+            if let Some(policies) = &self.policies {
```

Comment:
it looks like this is currently only checked for the "global" allow_credentials, not the origin-specific one, but this should also fail: 
```yaml
cors:
  allow_credentials: false
  policies:
    - { allow_any_origin: true, allow_credentials: true }
```

Reply:
Oh, or is `allow_any_origin` only allowed on the top level config? I guess then this works fine